.gitignore

```diff
@@ -35,7 +35,6 @@ coverage
 build/Release
 
 # Enterprise files
-server/main/imports.go
 server/enterprise
```

Comment:
I think this line and the comment are no longer needed.

Reply:
I was concerned someone might check in their left-over enterprise directory since this PR does not remove an existing sym-links.  I guess we would catch that in a review.